modules/learn/pages/services-and-indexes/indexes/index-replication.adoc

```diff
@@ -109,6 +111,8 @@ If index-maintenance is running behind, the query waits for it to catch up.
 For N1QL, the default consistency is `not_bounded`.
 // end::scan_consistency[]
 
+You can specify the scan consistency via the {query-preferences}[run-time preferences] in the Query Workbench, or by setting the {scan_consistency}[scan_consistency] request-level parameter.
+
 [[index-snapshots]]
 == Index Snapshots
 
```

Comment:
The snapshots is pointing to {database-change-protocol} which is not correct. These are index disk snapshots.

Reply:
Done